assets/__mocks__/popper.js.ts

```diff
@@ -0,0 +1,50 @@
+import StockPopperJs from 'popper.js';
+
+/*
+  From https://github.com/popperjs/popper-core/issues/478#issuecomment-638842453
+  Fixes an issue with popper.js running in jsdom:
+
+   PASS  test/SetExpiration.test.ts
+  ● Console
+
+    console.error
+      Warning: `NaN` is an invalid value for the `left` css style property.
+          in div (created by Tr)
+          in div (created by Tr)
+          in Tr (created by r)
+          in div (created by r)
+          in r (created by OnClickOutside(r))
+          in OnClickOutside(r) (created by r)
+          in div (created by InnerPopper)
+          in div (created by r)
+          in r (created by InnerPopper)
+          in InnerPopper (created by Context.Consumer)
+          in Popper (created by r)
+          in Manager (created by r)
+          in r (created by r)
+          in r (created by SetExpiration)
+          in div (created by SetExpiration)
+          in SetExpiration (created by WrapperComponent)
+          in WrapperComponent
```

Comment:
We probably don't need to include this full "element trace", especially since it would be different for every test.